app/src/main/java/com/yacgroup/yacguide/StatisticsActivity.kt

```diff
@@ -48,21 +48,34 @@ class StatisticsActivity : BaseNavigationActivity() {
     }
 
     private fun _displayContent() {
-        // Placeholders:
+        // The following counts contain all years, even if there was no ascent of the according style.
+        // This means that the list sizes are all equal.
```

Comment:
There should be some assertion in place that all those counts indeed contain the same years and are of the same length.

Reply:
How can this happen?

Reply:
It should not happen because of the way the SQL queries are constructed. However, as a user of this API, it's not really obvious that these lists are all of the same length with the same keys (year).